tests/cephfs/cephfs_mirroring/validate_file_dir_stats_modification_on_mirrored_cluster.py

```diff
@@ -40,6 +40,9 @@ def run(ceph_cluster, **kw):
         fs_util_ceph2.auth_list(target_clients)
         source_fs = "cephfs"
```

Comment:
can you create new fs other than cephfs. something like cephfs_dir_stats and cephfs_dir_stat_rem